surveys/2022-annual-survey/questions.md

```diff
@@ -645,21 +567,15 @@ Statements:
 - Rust makes me more productive
 - Rust is fun to use
 
-Rating:
```

Comment:
Are we sure we're not losing important signal by allowing people to actively disagree?

Reply:
IMO, it's a clearer signal because we don't let people cop out and choose the middle option. We might lose some signal here, but I think that simplifying for both the respondents and in the results is worth it